File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.js
function solution(my_string) {
    var answer = 0;
    var array = '';
    
    array = my_string.split("");
        
        for (i = 0; i < array.length; i++) {
            if(Number(array[i])) {
                answer += Number(array[i]);
            }
        }
    return answer;
}

File: 프로그래머스/0/120851. 숨어있는 숫자의 덧셈 （1）/숨어있는 숫자의 덧셈 （1）.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        
        String[] array = my_string.replaceAll("[^0-9]", "").split("");
        
        for (int i = 0; i < array.length; i++) {
            answer += Integer.parseInt(array[i]);
        }
        return answer;
    }
}